network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -104,7 +104,7 @@
         {% block background_parallax %}
         {% endblock %}
         <div class="tw-relative tw-z-10">
-            {% include "partials/footer.html" %}
+            {% include "fragments/buyersguide/pni_footer.html" %}
```

Comment:
```suggestion
            {% include "fragments/buyersguide/footer.html" %}
```


If we do end up renaming this file to `footer.html` this include statement will need to be updated too